cypress/support/scenarios/yves/checkout-scenario.ts

```diff
@@ -21,12 +21,13 @@ export class CheckoutScenario {
 
   execute = (params?: ExecuteParams): void => {
     this.cartPage.visit();
+      cy.wait(5000);
```

Comment:
all these waits should be removed. 
It's a general scenario that is executed N times. 3 waits by 5 seconds = +15 seconds just to place an order. Please remove them